databricks_cli/configure/provider.py

```diff
@@ -49,7 +49,14 @@ def _create_section_if_absent(raw_config, profile):
 
 
 def _get_option_if_exists(raw_config, profile, option):
-    return raw_config.get(profile, option) if raw_config.has_option(profile, option) else None
+    if profile == DEFAULT_SECTION:
```

Comment:
Seems we misunderstood the semantic of the `DEFAULT` section. I didn't expect the fallback behavior. Thanks for catching this!